httpobs/scanner/grader/grade.py

```diff
@@ -22,7 +22,28 @@
     0: 'F'
 }
 
+# See https://wiki.mozilla.org/Security/Standard_Levels for a definition of the risk levels
+# We cannot make an accurate decision on HIGH and MAXIMUM risk likelihood indicators with the current checks,
+# thus the likelihood indicator is currently at best (or worse) MEDIUM
+LIKELIHOOD_INDICATOR_CHART = {
+    'A+': 'LOW',
+    'A': 'LOW',
+    'A-': 'LOW',
+    'B+': 'MEDIUM',
+    'B': 'MEDIUM',
+    'B-': 'MEDIUM',
+    'C+': 'MEDIUM',
+    'C': 'MEDIUM',
+    'C-': 'MEDIUM',
+    'D+': 'MEDIUM',
+    'D': 'MEDIUM',
+    'D-': 'MEDIUM',
+    'F': 'MEDIUM'
+}
+
+
 GRADES = set(GRADE_CHART.values())
+LIKELIHOOD_INDICATOR = set(item for sublist in list(LIKELIHOOD_INDICATOR_CHART.values()) for item in sublist)
```

Comment:
it is used as a safe guard here:
https://github.com/mozilla/http-observatory/pull/110/files#diff-3322d6b3f471382a95cd3a089fcf4510R358

The syntax just follows prior syntax from the file (GRADES)

I can do:
if grade not in LIKELIHOOD_INDICATOR_CHART.keys():